app/concepts/cinema_halls/processors/modify_params.rb

```diff
@@ -0,0 +1,18 @@
+module CinemaHalls
+  module Processors
+    class ModifyParams
+      attr_reader :params, :seats
+
+      def initialize(params:, seats:)
+        @permit_params = params
```

Comment:
You have `:params` in attr_reader, and you don't use it

Reply:
You are right!